upi/openstack/inventory.yaml

```diff
@@ -4,8 +4,14 @@ all:
       ansible_connection: local
       ansible_python_interpreter: "{{ansible_playbook_python}}"
 
+      # User-provided values
       os_subnet_range: '10.0.0.0/16'
       os_cluster_name: 'openshift'
+      os_flavor_master: 'm1.xlarge'
+      os_image_rhcos: 'rhcos'
+
+      # Number of provisioned Control Plane nodes
+      os_cp_nodes_number: 2
```

Comment:
Oh sorry, I should have noticed this sooner. Control plane (masters) must have a minimum of three nodes.